Failure # 1 (occurred at 2025-05-22_22-13-35)
[36mray::ImplicitFunc.train()[39m (pid=92218, ip=127.0.0.1, actor_id=70def79efadf022a8032b0be01000000, repr=lstm_objective)
  File "/opt/homebrew/Caskroom/miniconda/base/lib/python3.13/site-packages/ray/tune/trainable/trainable.py", line 330, in train
    raise skipped from exception_cause(skipped)
  File "/opt/homebrew/Caskroom/miniconda/base/lib/python3.13/site-packages/ray/air/_internal/util.py", line 107, in run
    self._ret = self._target(*self._args, **self._kwargs)
                ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/miniconda/base/lib/python3.13/site-packages/ray/tune/trainable/function_trainable.py", line 45, in <lambda>
    training_func=lambda: self._trainable_func(self.config),
                          ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/miniconda/base/lib/python3.13/site-packages/ray/tune/trainable/function_trainable.py", line 261, in _trainable_func
    output = fn()
  File "/Users/mikeedwards/Library/Mobile Documents/com~apple~CloudDocs/pMike/Unity/price/training/objectives.py", line 19, in lstm_objective
    train_loader, val_loader = build_dataloaders(
                               ~~~~~~~~~~~~~~~~~^
        train_path=train_path,
        ^^^^^^^^^^^^^^^^^^^^^^
        val_path=val_path,
        ^^^^^^^^^^^^^^^^^^
        batch=64,
        ^^^^^^^^^
    )
    ^
  File "/Users/mikeedwards/Library/Mobile Documents/com~apple~CloudDocs/pMike/Unity/price/training/datamodule.py", line 34, in build_dataloaders
    train = PriceDataset(Path(train_path).resolve())
  File "/Users/mikeedwards/Library/Mobile Documents/com~apple~CloudDocs/pMike/Unity/price/training/datamodule.py", line 15, in __init__
    self.X, self.y = self.build_sliding_windows(df, seq_len)
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mikeedwards/Library/Mobile Documents/com~apple~CloudDocs/pMike/Unity/price/training/datamodule.py", line 24, in build_sliding_windows
    y.append(float(y_value))
             ~~~~~^^^^^^^^^
TypeError: float() argument must be a string or a real number, not 'Timestamp'
